app/mailers/smoke_test_prison_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class SmokeTestPrisonMailer < PrisonMailer
+  def self.smtp_settings
+    {
+      address: ENV['SMOKE_TEST_SMTP_HOSTNAME'],
+      port: ENV['SMOKE_TEST_SMTP_PORT']
```

Comment:
Do we need a corresponding `pvb-deploy` change to set these and set up pobox? Are you OK to do that, or do you want me to?